plugins/irma-web/dom-manipulations.js

```diff
@@ -173,12 +173,27 @@ module.exports = class DOMManipulations {
     `;
   }
 
-  _stateAborted() {
-    return `
-      <!-- State: Error -->
-      <div class="irma-web-forbidden-animation"></div>
-      <p>${this._translations.error}</p>
-    `;
+  _stateAborted(state) {
+    switch (state.oldState) {
```

Comment:
Ook hier zie ik best veel magic om de `abort` state te kunnen hergebruiken voor verschillende doeleinden. Ook dat roept de vraag op of dit niet meerdere states hadden moeten zijn.

Reply:
Ik denk dat ik deze logica nog wel vrij ingrijpend kan versimpelen gezien jouw proposal om het renderen van de `Abort` state te versimpelen in de `abort-state` pull request. Daar ga ik morgen nog wel even een poging in doen. Ik hoop dat ik het nog wat mooier kan krijgen zonder nog meer extra states te hoeven introduceren. Ik denk dat dat moet kunnen.